packages/docusaurus-theme-bootstrap/src/theme/BlogPostPage/index.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
```

Comment:
I think it'd be better to use empty components for them. Just add `export default () => {};`. Same for the rest of the empty components.

Reply:
Agree